.changeset/angry-months-pay.md

```diff
@@ -6,11 +6,13 @@ Adopt Remix [`v2_meta`](https://remix.run/docs/en/main/route/meta#metav2) future
 
 ### `v2_meta` migration steps
 
-1. For any routes that you used `meta` route export, convert it to the `V2_MetaFunction` equivalent.
+1. For any routes that you used `meta` route export, convert it to the `V2_MetaFunction` equivalent. Notice that the package name in the import statement has also changed to `'@remix-run/react'`:
 
    **Before** - returns an object;
 
    ```jsx
+   import {type MetaFunction} from '@shopify/remix-oxygen';
```

Comment:
Should we consider using the `diff` block here, to make it a lot easier to see what changes need to happen?